ten_quick_tips_live_coding.md

```diff
@@ -22,8 +22,8 @@ or relying on learners reading a textbook or compendium.
 What is taught is immediately applied rather than just shown on a slide or on paper.
 It also slows the instructor down,
 giving learners more time to actively engage with the material before moving on to the next concept.
-The thought process behind coding also becomes more apparent when this is observed live.
-Learner's questions can immediately be answered and misconceptions corrected by demonstration.
+Importantly, the thought process behind coding can also be made explicit.
+Learner's questions can immediately be answered and misconceptions corrected by coding them.
```

Comment:
Worth adding the analogy that slides are like riding in a train while live coding is like driving in a car? Train is a smoother journey but you've decided the route in advance and can't change it; car might be a little bumpier, but you can explore.